activesupport/lib/active_support/deprecation/proxy_wrappers.rb

```diff
@@ -25,15 +25,19 @@ def method_missing(called, *args, &block)
         end
     end
 
-    # This DeprecatedObjectProxy transforms object to deprecated object.
+    # DeprecatedObjectProxy transforms an object into a deprecated object. It takes an object,
+    # a deprecation message, and optionally a deprecator. The deprecator defaults to
+    # <tt>ActiveSupport::Deprecator</tt> if none is specified.
     #
-    #   @old_object = DeprecatedObjectProxy.new(Object.new, "Don't use this object anymore!")
-    #   @old_object = DeprecatedObjectProxy.new(Object.new, "Don't use this object anymore!", deprecator_instance)
+    #  Example:
```

Comment:
We avoid the work `Example:` in our documentation see http://guides.rubyonrails.org/api_documentation_guidelines.html#example-code